browser/ui/commander/bookmark_command_source.cc

```diff
@@ -0,0 +1,89 @@
+// Copyright 2020 The Chromium Authors
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
```

Comment:
We should update all of these header comments so the linter doesn't trip over them.

Reply:
Ah yeah, so I wasn't sure how to handle those - I don't know what we normally do, but I think the copyright probably still belongs to The Chromium Authors?